addons/payment_stripe/models/payment_transaction.py

```diff
@@ -55,11 +55,11 @@ def _stripe_create_checkout_session(self):
         :rtype: dict
         """
         # Filter payment method types by available payment method
-        existing_pms = [pm.name.lower() for pm in self.env['payment.icon'].search([])]
-        linked_pms = [pm.name.lower() for pm in self.provider_id.payment_icon_ids]
+        existing_pms = [pm.name.lower() for pm in self.env['payment.method'].search([])]
+        linked_pms = [pm.name.lower() for pm in self.provider_id.payment_method_ids]
         pm_filtered_pmts = filter(
             lambda pmt: pmt.name == 'card'
-            # If the PM (payment.icon) record related to a PMT doesn't exist, don't filter out the
+            # If the PM (payment.method) record related to a PMT doesn't exist, don't filter out the
             # PMT because the user couldn't even have linked it to the provider in the first place.
```

Comment:
The parenthesis is no longer needed.
```suggestion
            # If the PM record related to a PMT doesn't exist, don't filter out the PMT because the
            # user couldn't even have linked it to the provider in the first place.
```